### DISCUSSION TITLE
pcap through SO

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#5350] [General] [answered]
Any possibilities of generating pcap centrally from SO? I need pcap for datasets which I need to collect individually from host machines.

**Comment by dougburks** [comment, marked as answer]
I'm not sure I understand what you're asking.

Have you checked the documentation?

https://docs.securityonion.net/en/2.3/pcap.html

https://docs.securityonion.net/en/2.3/stenographer.html
